config.js

```diff
@@ -46,7 +46,9 @@ config.server = {
   aggregationBy: ['day', 'hour', 'minute'],
   // Directory where temporary files will be stored, such as the ones generated when CSV files are requested.
   // Default value: "temp".
-  temporalDir: 'temp'
+  temporalDir: 'temp',
+  // Max page size returned by a query
+  maxPageSize: '100'
```

Comment:
New parameters in config.js should be documented. Does it has an associated env var (also to be documented, I guess)?

Reply:
Fixed in https://github.com/telefonicaid/fiware-sth-comet/pull/437/commits/3d9ac11fba7339d129a66c56cebc56c2864f148a